Update pom.xml

machphy opened the pull request:
Changes Made:
Version Update: Updated the maven-clean-plugin to version 3.3.1.
Verbose Logging: Added <verbose>true</verbose> to enable detailed logging during the clean phase.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/questdb/questdb?pullRequest=4601) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/questdb/questdb?pullRequest=4601) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/questdb/questdb?pullRequest=4601) it.</sub>

machphy commented:
Changes Made:
Version Update: Updated the maven-clean-plugin to version 3.3.1.
Verbose Logging: Added <verbose>true</verbose> to enable detailed logging during the clean phase.
